recipes-samples/images/rpb-weston-image.bb

```diff
@@ -15,6 +15,7 @@ CORE_IMAGE_BASE_INSTALL += " \
     sshfs-fuse hostapd iptables \
     alsa-utils-aplay gstreamer1.0-plugins-bad-meta gstreamer1.0-plugins-base-meta gstreamer1.0-plugins-good-meta \
     ${@bb.utils.contains("MACHINE_FEATURES", "optee", "optee-test optee-client", "", d)} \
+    ${@bb.utils.contains("MACHINE_FEATURES", "mali450", "mali450-userland", "", d)} \
```

Comment:
On 06/29/2016 11:56 PM, Nicolas Dechesne wrote:
> In recipes-samples/images/rpb-weston-image.bb <https://github.com/96boards/meta-rpb/pull/10#discussion_r69026124>:
>
>> @@ -15,6 +15,7 @@ CORE_IMAGE_BASE_INSTALL += " \
>>      sshfs-fuse hostapd iptables \
>>      alsa-utils-aplay gstreamer1.0-plugins-bad-meta gstreamer1.0-plugins-base-meta gstreamer1.0-plugins-good-meta \
>>      ${@bb.utils.contains("MACHINE_FEATURES", "optee", "optee-test optee-client", "", d)} \
>> +    ${@bb.utils.contains("MACHINE_FEATURES", "mali450", "mali450-userland", "", d)} \
>
> the way I solved that for meta-qcom is
>
> MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'mesa-driver-msm', '', d)}"
>
> i am not saying it's better..

Looks promising imho.
(That is I like, but just need some more time to understand how and when 'opengl' is added to
DISTRO_FEATURES)

> but considering mali450 a MACHINE_FEATURES looks weird to me. maybe it's been discussed

- a little bit

> and agreed already, in which case it's fine..

My hope is that the current version is OK for now. I could get back to your solution afterwards.

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/96boards/meta-rpb/pull/10/files/e5b7194617d61d814ad21770557697716e88a6fc..edc3e99f45605690d608b15b8c82aa726ed83ad3#r69026124>, or mute the thread
> <https://github.com/notifications/unsubscribe/APSqealkrKLw4B99tnk5ZsWyZUfj8xemks5qQtwCgaJpZM4JBYYO>.
>